### PR TITLE
Added the option for having fixed label to the directive

### PR DESCRIPTION
**Pull request by dassaptorshi**
If possible please merge it to 5.4.3 as we have started using that version of yours.

Please kindly approve

### CODE REVIEW

**Comment by ValentinH** [comment]
Please describe the purpose of this PR and provide a demo.

**Comment by dassaptorshi** [comment]
This PR is to configure the directive so the limit labels don't hide when we reach a particular limit towards floor or ceil value.

[Demo](http://plnkr.co/edit/nOf5YX?p=info)

**Comment by dassaptorshi** [comment]
Hi Valentin,
How long do you think it may take to  merge the pull request?

Thanks in advance

File: src/rzslider.js [hunk 1]
```diff
@@ -1093,8 +1096,10 @@
             backgroundColor: pointercolor
           };
         }
-
-        this.shFloorCeil();
+        if(!this.optionsshowLimitLabels){
```
Comment (ValentinH) [inline src/rzslider.js]: this is wrong

File: dist/rzslider.js [hunk 2]
```diff
@@ -1089,8 +1092,11 @@
             backgroundColor: pointercolor
           };
         }
-
-        this.shFloorCeil();
+        console.log(this.options.showLimitLabels)
```
Comment (ValentinH) [inline dist/rzslider.js]: this should be removed.

**Comment by ValentinH** [comment]
I'm sorry, I'm currently on holidays so I don't have access to my computer and won't be able to release it anyway...

Here are a few remarks:
-  I would prefer to replace the `showLimitLabels` by something like `autoHideLabels: true` by default so its purpose is more obvious.
- other comments are on the lines directly...

**Comment by dassaptorshi** [comment]
Done!

**Comment by ValentinH** [comment]
The dist file is still incorrect, you should run grunt before commiting...

**Comment by dassaptorshi** [comment]
Done!

**Comment by ValentinH** [comment]
Merged and release under 5.5.0. I have just rename the new option to `autoHideLimitLabels` since it makes more sense.

**Pull request merged**